gcloud-java-logging/src/main/java/com/google/cloud/logging/spi/v2/ConfigServiceV2Api.java

```diff
@@ -256,103 +210,92 @@ public void close() throws IOException {
     }
   }
 
-  // ----- listSinks -----
-
-  // AUTO-GENERATED DOCUMENTATION AND METHOD - see instructions at the top of the file for editing.
+  // AUTO-GENERATED DOCUMENTATION AND METHOD
   /**
    * Lists sinks.
    *
    * Sample code:
    * <pre><code>
-   * try (ConfigServiceV2Api configServiceV2Api = ConfigServiceV2Api.createWithDefaults()) {
-   *   String formattedProjectName = ConfigServiceV2Api.formatProjectName("[PROJECT]");
-   *   for (LogSink elements : configServiceV2Api.listSinks(formattedProjectName)) {
-   *     // doThingsWith(elements);
+   * try (ConfigServiceV2Api configServiceV2Api = ConfigServiceV2Api.create()) {
+   *   String formattedParent = ConfigServiceV2Api.formatProjectName("[PROJECT]");
```

Comment:
Should `formatProjectName` be renamed to `formatParentName`? If we do this all examples must be updated accordingly.